src/Tasks/Engines/DoormanRunner.php

```diff
@@ -47,6 +48,13 @@ public function getDefaultRules()
      */
     public function runQueue($queue)
     {
+        // check if queue can be processed
+        $service = QueuedJobService::singleton();
+        if ($service->isAtMaxJobs()) {
+            $service->getLogger()->error('Not processing queue as jobs are at max initialisation limit.');
```

Comment:
Note that #244 reduced this to an `info` level. I have just merged those branches up so, so you'll probably get conflicts in the other method refactor you did above, but maybe you could use `->info()` here instead?